requirements.dev

```diff
@@ -0,0 +1,16 @@
+beautifulsoup4
```

Comment:
remove beautifulsoup?

Reply:
Actually, turns out we do still use it in `readers.py`. Sorry!

Reply:
Based on the type annotations, it cannot be bytes `file: IO[str]` :)
https://github.com/EducationalTestingService/skll/blob/affb97a658e9f9e679ada0fe530f9dccb84f9b5a/skll/data/readers.py#L868

I assume `bs4.UnicodeDammit` cleverly guesses the encoding(?) - could maybe have a keyword argument for the encoding to get rid of bs4.

Reply:
Yep, we used it for converting to the encoding we needed since we used to deal with a lot of weirdly encoded files back in the day. May be we don't need it anymore but I don't want to investigate that in this MR.